src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -85,22 +87,18 @@ describe('MdAutocomplete', () => {
           .toContain('California', `Expected panel to display when opened programmatically.`);
     });
 
-    it('should close the panel when a click occurs outside it', async(() => {
+    it('should close the panel when blurred', async(() => {
```

Comment:
You mean when you make the  md-option active by the keyboard, right? Focus never changes to md-option unless you click on it, in which case the panel should close.